libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -248,3 +248,22 @@ void SRV_Channels::push()
     }
 #endif // HAL_WITH_UAVCAN
 }
+
+// return a bitmask of each channel that is a motor or a throttle
+uint16_t SRV_Channels::get_motor_mask()
+{
+    uint16_t motor_bitmask = 0;
```

Comment:
Add a static assert that `NUM_SERVO_CHANNELS` <= 16, otherwise this could be a problem in the future.

Reply:
done